curvefs/src/mds/topology/topology_item.h

```diff
@@ -55,6 +55,8 @@ struct ClusterInformation {
     std::string clusterId;
     // <fsId, partition index of this fs>
     std::map<uint32_t, uint32_t> partitionIndexs;
+    // <fsId, next inode id>
+    std::unordered_map<uint32_t, uint64_t> fsMaxAllocatedInodeId;
```

Comment:
How about `fsNextInodeId`? And there are also some related functions that can be modified according to this.

Reply:
fixed